op3_ball_detector/src/ball_detector_config.cpp

```diff
@@ -31,4 +31,12 @@ int BallColorConfig::getMedianRVal(int x_val)
     return light_slope * x_val + light_constant;
 }
 
+void BallColorConfig::updateDistribution(std::vector<double> light_range, std::vector<double> range_weights)
+{
```

Comment:
Is it safe to assume these will be developed this week?  If not, I think we should remove them before feature freeze when we leave on thursday.  I'm still comfortable approving the merge, I'm just keeping things in perspective from a high level view

Reply:
This is what Mel is implementing / has implemented on his branch. I just defined the stubs to make the merging easier, and so my code can compile as is.

Reply:
Perfect thanks for the heads up =)